datafusion/core/tests/sqllogictests/test_files/math.slt

```diff
@@ -87,3 +87,24 @@ query RRRRRRRR
 SELECT round(125.2345, -3), round(125.2345, -2), round(125.2345, -1), round(125.2345), round(125.2345, 0), round(125.2345, 1), round(125.2345, 2), round(125.2345, 3)
 ----
 0 100 130 125 125 125.2 125.23 125.235
+
+# Create table for atan2 test
+statement ok
```

Comment:
its good to drop table after the test completed, or as alternative you can prepare data in WITH clause.

Reply:
Thank you for the feedback, I think even creating a table is not necessary. Updated